### PR TITLE
Global footer/madjouga

### PR DESCRIPTION
**Pull request by amadjouga**
Verify that component of Global Footer is displayed on the bottom
Verify that component of Global Footer is displayed on a page with two lines.
Verify that fist line include “© {year} Bug Tracker by React Web Development”
Verify that second line include “Version: {version}”
Verify that {year} = current year.
Verify that {version} = current version.
Verify the version release (currently 0)
Verify the Sprint # (currently 3)
Verify third digit gets increased every code change(pending)

### CODE REVIEW

File: test/selectors/global-footer.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default {
+    footer: '.custom-footer',
+    text: '//*[@id="root"]/div/div[3]/div/div',
```
Comment (artsenius) [inline test/selectors/global-footer.js]: Use the same approach as @AlexB-447 did during the lesson. You can use <div> tag for first line and <span> for the seconf one.

Comment (artsenius) [reply test/selectors/global-footer.js]: Resolve conflict

File: test/specs/reg/global-footer/general.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+describe('General', function () {
+
+    it('Footer id displayed', function () {
+        browser.url('/');
+        let footer = $(sel.footer).isDisplayed();
+        assert.isTrue(footer);
+    });
+    it('Two lines text', function () {
+        let line_1 = $(sel.text).isDisplayed();
+        let line_2 = $(sel.smalltext).isDisplayed();
```
Comment (artsenius) [inline test/specs/reg/global-footer/general.js]: You can use isVisible method from helpers

Comment (amadjouga) [reply test/specs/reg/global-footer/general.js]: @artsenius Please check my changes.

**Pull request merged**